src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -155,11 +156,12 @@ slot[name='label'] > span {
 
 // Input
 /* stylelint-disable plugin/stylelint-bem-namics */
-.form-field__input ::slotted(:where(input, select, .sbb-form-field-element)) {
+.form-field__input ::slotted(:where(input, select, [data-sbb-form-field])) {
   @include text-m--regular;
   @include placeholder;
   @include ellipsis;
 
+  overflow: var(--form-field-overflow);
```

Comment:
I'm just interested, which is the use case where an overflow hidden is necessary?

Reply:
overflow hidden comes from the @include ellipsis, it's necessary for ellipsis to work
on the other side (slider) this instruction hides part of the knob

Reply:
ah I see now what confused me, it was the new assignment on line 24. `--form-field-overflow: hidden;`. because you override the mixins value, it's necessary to redefine it. It's ok.